llvm/include/llvm/CodeGen/SDPatternMatch.h

```diff
@@ -226,6 +226,15 @@ inline SwitchContext<MatchContext, Pattern> m_Context(const MatchContext &Ctx,
   return SwitchContext<MatchContext, Pattern>{Ctx, std::move(P)};
 }
 
+struct m_Mask {
```

Comment:
this is not used anymore I believe

Reply:
As the capture logic is moved inside `SDShuffle_match`, I think removing this is correct.